zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/repo/VFSNotebookRepoTest.java

```diff
@@ -45,22 +45,21 @@ class VFSNotebookRepoTest {
   private ZeppelinConfiguration zConf;
   private NoteParser noteParser;
   private VFSNotebookRepo notebookRepo;
-  private File notebookDir;
```

Comment:
I removed this field. I think the notebookRepo.rootNotebookFolder would be enough to check notebook directory.
This is also related to the issue that a temp directory was created every time [this test](https://github.com/apache/zeppelin/pull/4789/files#diff-3dde8f55d9baf47750cc307761685baf82d85588e042984b181baa32893e9552R144) is run. I think it was caused by the directory path data being managed by multiple variables.

Reply:
Yes, I agree